pages/wallets/index.tsx

```diff
@@ -34,14 +36,53 @@ export const getServerSideProps: GetServerSideProps = async (context) => {
   }
 }
 
-interface PaybuttonsProps {
+interface WalletsProps {
   userId: string
 }
 
-export default function Wallets ({ userId }: PaybuttonsProps): React.ReactElement {
+interface WalletsState {
+  wallets: [{name: string, paybuttons: any[], xec_balance: string, bch_balance: string, payments: string}]
+}
+
+export default function Wallets ({ userId }: WalletsProps): React.ReactElement {
   return (
     <ThirdPartyEmailPasswordAuthNoSSR>
-      <h2>Wallets</h2>
+      <ProtectedPage userId={userId} />
     </ThirdPartyEmailPasswordAuthNoSSR>
   )
 }
+
+class ProtectedPage extends React.Component<WalletsProps, WalletsState> {
+  constructor (props: WalletsProps) {
+    super(props)
+    this.state = {
+      wallets: [
+        {
+          name: 'Default Wallet',
```

Comment:
We should have an actual field here for 'default_wallet' or whatever which is a true/false field.

Can we also have some sort of indicator somewhere that tells people that this is the default wallet (even if they change the wallet name)?